modules/hostap/src/supp_main.c

```diff
@@ -306,9 +306,32 @@ static void iface_event_handler(struct net_mgmt_event_callback *cb,
 							uint32_t mgmt_event, struct net_if *iface)
 {
 	const char *ifname = iface->if_dev->dev->name;
+	char *tmp_csv;
+	char *prefix;
+	char *mgd_if_prefix = CONFIG_WPA_SUPP_MGD_IFACES_PREFIX;
+	bool found = false;
+
+	if (strlen(mgd_if_prefix) > 0) {
+		tmp_csv = os_zalloc(strlen(mgd_if_prefix));
```

Comment:
We do not need this heap allocation anymore, as we can manipulate the `mgd_if_prefix` variable as we want.

Reply:
Done